app/controllers/users_controller.rb

```diff
@@ -76,6 +76,8 @@
 class UsersController < ApplicationController
   before_action :authenticate
   before_action :correct_user, only: [:edit, :update]
+  before_action :active_user, only: [:index]
```

Comment:
Purely style, but given that these only apply to a single action would it be easier to get rid of the `before_action` and the helper method and just call the logic directly?

Reply:
Possibly, but since all of these are related to user permissions it does keep all that logic in one place.  Hopefully it will make it easier to migrate us out to a halfway decent permissions solution later.

Reply:
yeah makes it easier to expand if we were to add `:delete` or `:give_free_puppy` or something down the line.